src/ticketing/types.ts

```diff
@@ -193,3 +193,11 @@ export type AddPaymentMethodResponse = {
   recurring_payment_id: number;
   terminal_url: string;
 };
+
+/**
+ * Defined by RefundOptionsResponse in
+ * https://github.com/AtB-AS/sales/blob/main/sales-service/src/handlers/sales/refund.rs#L31
+ */
```

Comment:
Good to have comments like this :+1:

Reply:
I guess the line number will change over time though, so removing that for now ✌️

```suggestion
/**
 * Defined by RefundOptionsResponse in
 * https://github.com/AtB-AS/sales/blob/main/sales-service/src/handlers/sales/refund.rs
 */
```